### PR TITLE
set :fatal to duplicated_fqdns by default

### PR DESCRIPTION
**Pull request by sawanoboly**
Follow this feature https://github.com/chef/chef/pull/6781

### CODE REVIEW

File: lib/chef/knife/zero_converge.rb [hunk 1]
```diff
@@ -20,6 +20,10 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # force fatal
+      self.options.delete :duplicated_fqdns
+      self.options[:duplicated_fqdns] = :fatal
```
Comment (sawanoboly) [inline lib/chef/knife/zero_converge.rb]: __[RuboCop]__

Style/RedundantSelf: Redundant `self` detected.

File: lib/chef/knife/zero_converge.rb [hunk 2]
```diff
@@ -20,6 +20,10 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # force fatal
+      self.options.delete :duplicated_fqdns
```
Comment (sawanoboly) [inline lib/chef/knife/zero_converge.rb]: __[RuboCop]__

Style/RedundantSelf: Redundant `self` detected.

File: lib/chef/knife/zero_converge.rb [hunk 3]
```diff
@@ -20,6 +20,12 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # force fatal Ref: https://github.com/chef/chef/pull/6781
+      if self.options[:duplicated_fqdns]
+        self.options.delete :duplicated_fqdns
+        self.options[:duplicated_fqdns] = :fatal
```
Comment (sawanoboly) [inline lib/chef/knife/zero_converge.rb]: __[RuboCop]__

Style/RedundantSelf: Redundant `self` detected.

File: lib/chef/knife/zero_converge.rb [hunk 4]
```diff
@@ -20,6 +20,11 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # fatal by default Ref: https://github.com/chef/chef/pull/6781
+      if self.options[:duplicated_fqdns]
```
Comment (sawanoboly) [inline lib/chef/knife/zero_converge.rb]: __[RuboCop]__

Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

File: lib/chef/knife/zero_converge.rb [hunk 5]
```diff
@@ -20,6 +20,12 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
...
+        self.options[:duplicated_fqdns][:description] = 'Behavior if FQDNs are duplicated, fatal by default'
```
Comment (sawanoboly) [inline lib/chef/knife/zero_converge.rb]: __[RuboCop]__

Metrics/LineLength: Line is too long. [108/100]

**Comment by linyows** [comment]
Thank you for chef following!

**Comment by sawanoboly** [comment]
Released as v1.19.2.

**Pull request merged**